docs/user-guide/dual-mesh.ipynb

```diff
@@ -0,0 +1,6253 @@
+{
```

Comment:
<p>We should include a short overview/description between the Title and Imports to be consistent with the other user guide sections.</p><p> </p><p>For example, <a href="https://uxarray--859.org.readthedocs.build/en/859/user-guide/data-structures.html#data-structures" target="_blank">https://uxarray--859.org.readthedocs.build/en/859/user-guide/data-structures.html#data-structures</a></p><p> </p><p> </p><p>Also, we use the term "Dual Mesh" and "Dual Grid" pretty interchangeably, but for consistency let's always refer to it as either the "Dual" or a "Dual Grid" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/859/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>